### DISCUSSION TITLE
Uncertainty, SLR & Tides Across StormSim

### DISCUSSION THREAD

**Discussion by fgarcia0128** [coastal-hazards-system/StormSim-Library#14] [General]
This is meant to discuss how we will address uncertainty, SLR & tides across our workflows. Forcing data adjustments should be done outside computational routines. Forcing adjustments included in the software so far include:
- SLR (fixed value)
- Tides (Random tide applied to SWL)
- Uncertainty (PCHA methodology U_a, U_r)

@jmelby @NCNC-628 @KHodgens

**Comment by jmelby** [comment]
There are hard-coded tide and SLR statements within JPM and SST codes.  Also note that SLR applied within life cycle simulation will result in nonstationary time series.  This will require special care when doing statistical analysis.

**Comment by KHodgens** [comment]
Can we state the required assumptions (e.g., stationarity) up front for CHART?  Meaning that any stats will operate on unadulterated events.  I'm not sure what that means for random tides and replicates.
